pkg/cb/cbdsl/events.go

```diff
@@ -0,0 +1,116 @@
+package cbdsl
+
+import (
+	"github.com/filecoin-project/mir/pkg/modules/dsl"
+	"github.com/filecoin-project/mir/pkg/pb/cbpb"
+	"github.com/filecoin-project/mir/pkg/pb/eventpb"
+	"github.com/filecoin-project/mir/pkg/pb/messagepb"
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+// Contains event wrappers specific for this module
+
+func SignRequest(m *cbModuleImpl, destModule t.ModuleID, data [][]byte) {
```

Comment:
This might be a matter of personal taste, but it could be more intuitive to have this function as well as the other ones in this file be methods of m.

Reply:
The examples aren't ideal because they operate on `cbdsl.cbModuleImpl` and can be methods of this type. However, if there is a package that defines a new event type (assuming one day we will make it possible to define event types outside the framework itself), then the `UponEvent(m, ...)`  and `Event(m, ...)` should be defined for general `DslModule` and in that package. This would not be possible for methods.

Reply:
Aesthetically, I would also prefer using methods :)